docs/public-networks/reference/cli/options.md

```diff
@@ -4784,6 +4784,13 @@ The synchronization mode. Use `SNAP` for [snap sync](../../get-started/connect/s
 - The default is `FULL` when connecting to a private network by not using the [`--network`](#network) option and specifying the [`--genesis-file`](#genesis-file) option.
 - The default is `SNAP` when using the [`--network`](#network) option with named networks, except for the `dev` development network. `SNAP` is also the default if running Besu on the default network (Ethereum Mainnet) by specifying neither [network](#network) nor [genesis file](#genesis-file).
 
+:::note Sync nodes for BFT
+
+If you're using a Byzantine Fault Tolerance (BFT) consensus mechanism such as QBFT or IBFT 2.0, set your node's [synchronization mode](/public-networks/get-started/connect/sync-node#run-a-full-node) to either `FAST` or `FULL`. 
+By default, the option is set to `--sync-mode=FAST`.
```

Comment:
```suggestion
If you're running a node on a [QBFT](/private-networks/how-to/configure/consensus/qbft) or [IBFT 2.0](/private-networks/how-to/configure/consensus/ibft) network, you must use fast sync or full sync. 
```

Simplify as per my previous comments and make sure you take in the context of where this note is placed. The sync Besu page is already linked throughout this option description, and defaults are already described, so no need to add additional links or wordiness in this note.

Reply:
:::note Sync nodes for BFT

If you're running a node with the consensus mechanism IBFT 2.0 or QBFT, your node must use either fast sync or full sync. 

:::